.travis.yml

```diff
@@ -17,6 +17,12 @@ env:
   - EXPECTED_VERSION=10.0.2 JDKW_DIST=zulu JDKW_BUILD=10.3+5 JDKW_VERSION=10.0.2 JDKW_VERBOSE=1
   - EXPECTED_VERSION=11.0.3 JDKW_DIST=zulu JDKW_BUILD=11.31.11 JDKW_VERSION=11.0.3 JDKW_VERBOSE=1
   - EXPECTED_VERSION=12.0.1 JDKW_DIST=zulu JDKW_BUILD=12.2.3 JDKW_VERSION=12.0.1 JDKW_VERBOSE=1
+  - EXPECTED_VERSION=1.8.0_232 JDKW_DIST=adpt JDKW_BUILD=b09 JDKW_VERSION=8u232 JVM_IMPL=hotspot JDKW_VERBOSE=1
```

Comment:
Unless `adpt` is a common short-hand for Adopt JDK, I would prefer we use `adopt` as the identifier and reflect that in the code as well.

Reply:
Changed every occurence of adpt to adopt